dev-python/pyenchant/pyenchant-3.2.0.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Python bindings for the Enchant spellchecker"
+DESCRIPTION="This package provides a set of Python language \
+bindings for the Enchant spellchecking library. For more \
+information, visit the project website."
+HOMEPAGE="http://pyenchant.github.io/pyenchant/"
+COPYRIGHT="2020 Dimitri Merejkowsky"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/pyenchant/pyenchant/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="3195bfc5cfbdeb8a60f9484390c2c982e59dd220d6f9a5f8418ba67551ead01a"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libenchant_2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+PYTHON_PACKAGES=(python3 python38 python39)
+PYTHON_VERSIONS=(3.7 3.8 3.9)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+pythonPackage=${PYTHON_PACKAGES[i]}
+pythonVersion=${PYTHON_VERSIONS[$i]}
+eval "PROVIDES_${pythonPackage}=\"\
+	${portName}_$pythonPackage = $portVersion\
+	\"; \
+REQUIRES_$pythonPackage=\"\
+	haiku\n\
+	cmd:python$pythonVersion\
```

Comment:
The base package now only has a requirement for libenchant_2, do the subpackages require that too?
If so, maybe move this here as a requirement?

Reply:
pyenchant is a python wrapper for enchant, so all packages need it.